src/views/bill/_search.php

```diff
@@ -74,7 +74,7 @@
 
 
 <div class="col-md-4 col-sm-6 col-xs-12">
-    <?= $search->field('servers') ?>
+    <?= $search->field('server_ilike') ?>
```

Comment:
It will include nicX devices, I'm not sure this change is desired.

@hiqsol what's your opinion?

Reply:
They can exclude nic devices with other filter.
If they require comparable searches in two panels obviously they need same filters.
So 2 options:
- convince them that they will not receive comparable searches
- create same filters somehow